### PR TITLE
fix(msp): fix defects related to active monitoring

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
1. The URL cannot be modified when repairing editing
2. When repairing the selection of status code and request body, the data of operate is not displayed normally
## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     1.  The URL cannot be modified when repairing editing 2.  When repairing the selection of status code and request body, the data of operate is not displayed normally        |
| 🇨🇳 中文    |     1.   修复编辑时无法修改URL 2.   切换状态码和请求体的时候operate没有正常显示     |

## Does this PR need be patched to older version?
✅ Yes(version is required)
release/1.4
release/1.4.1

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx [hunk 1]
```diff
@@ -298,7 +290,9 @@ const AddModal = (props: IProps) => {
     switch (bodyType) {
       case noneType:
         updater.textOrJson('');
-        updater.headers({});
+        updater.headers({
+          ...headers,
```
Comment (Zero-Rock) [inline shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:294]: What's the point of this

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx [hunk 2]
```diff
@@ -242,7 +234,7 @@ const AddModal = (props: IProps) => {
         projectId,
         tenantId: terminusKey,
         config: {
-          url,
+          url: formRef.current?.getFieldValue(['config', 'url']),
```
Comment (Zero-Rock) [inline shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:237]: can use `_data.config.url`

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx [hunk 3]
```diff
@@ -263,7 +255,7 @@ const AddModal = (props: IProps) => {
           retry,
           interval: frequency,
           headers,
-          url,
+          url: formRef.current?.getFieldValue(['config', 'url']),
```
Comment (Zero-Rock) [inline shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:258]: same as above

Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx]: modified

File: shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx (hunk 1 above)
Comment (wuhuiluo) [reply shell/app/modules/msp/monitor/status-insight/pages/status/add-modal.tsx:294]: When there is a value in the headers and the type of the body is switched to none, the value of the original headers remains unchanged

**Pull request merged**